Sources/SwiftGen/Logs.swift

```diff
@@ -46,9 +46,9 @@ func logMessage(_ level: LogLevel, _ string: CustomStringConvertible) {
   case .info:
     fputs(ANSIColor.green.format("\(string)\n"), stdout)
   case .warning:
-    fputs(ANSIColor.yellow.format("Warning: \(string)\n"), stderr)
+    fputs(ANSIColor.yellow.format("swiftgen: warning: \(string)\n"), stderr)
```

Comment:
This works? (not having error/warning at the start of the line)

Reply:
Not only does it work but it's part of the expected format:

`<filename>:<line>:<col>: warning: <message>`

Where the `<col>:` can be (and often is) omitted, and if it is,`<line>:` can be omitted too.
`<filename>:` can _also_ be omitted completely, but it's less common; and `warning:`/`error:` are lowercase, even if maybe Xcode might be case insensitive about that to trigger a yellow/red banner on the file/line

Reply:
👍

Reply:
Note that for a minute there I thought about changing the enum (.info/.warning/.error) to have associated values for file and line. Then realized for config file we don't have line references back in the original YAML. And I wanted to do a quick PR that we could merge in 6.0.1 fast, while doing such a change would affect quite a lot in the codebase. So in the end I chose to keep it simple

In the ideal solution we wouldn't just print `swiftgen:` at the beginning of that log line but `swiftgen.yml` if the error is coming from the (default) config file and potentially something else if it's become from some other source (like error parsing an input storyboard or whatnot), hence the idea of an enum with associated value

But didn't want to use `swiftgen.yml:` in the generic log message here because until we move to an EAV we are not sure if the error comes from another place other than the config file or if the config file used isn't the default one and have a different name… so I kept all that for later and kept it simple